packages/constellation/src/components/Base/Colors/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Colors } from './Colors'
```

Comment:
Does it make sense to be exporting the color component from the color palette gallery? To me, this is a nice feature of viewing the component library, but not something that I would want to import as a consumer of the library.

What would be useful as a consumer would be an exported enum of all semantic colors and their css var name so that I can pass these to components like the icon.

Rough Eg 
```
import Colors from 'constellation'
...
<IconPlus color={Colors.Primary} />
```

What do you think?

Reply:
Yeah I don't think it is worth exporting any of the components that currently exist there as they really wouldn't be helpful in a consuming app.

This is probably a good base for exporting enums though yeah as I could modify the current util that supplies the info to the story components.